taxcalc/tests/test_functions.py

```diff
@@ -0,0 +1,40 @@
+"""
+Tests for Tax-Calculator functions.py file.
+"""
+# CODING-STYLE CHECKS:
+# pep8 --ignore=E402 test_functions.py
+# pylint --disable=locally-disabled test_functions.py
+# (when importing numpy, add "--extension-pkg-whitelist=numpy" pylint option)
+
+import os
+import sys
+import re
+CUR_PATH = os.path.abspath(os.path.dirname(__file__))
+
+
+def test_function_args_usage():
+    """
+    Checks each function argument in functions.py for use in its function body.
+    """
+    funcfilename = '{}/../functions.py'.format(CUR_PATH)
```

Comment:
something like:
```
os.path.join(CUR_PATH, '..', 'functions.py')
```
is probably best and is cross-platform (handles `\` for path separator for Windows platform